libc/test/src/__support/CPP/array_test.cpp

```diff
@@ -28,6 +28,11 @@ TEST(LlvmLibcArrayTest, Basic) {
   ASSERT_EQ(*(++it), 1);
   ASSERT_EQ(*(++it), 0);
 
+  auto forward_it = a.begin();
```

Comment:
This tests `cpp::array` and not `cpp::fixed_vector`.

Reply:
Ah, I totally made a mistake